### DISCUSSION TITLE
Unveiling the New Identity. Bloatynosy or X

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#120] [Announcements]
Here is a stable preview under the new but still temporary name "Appstrip". The voting for the app name is still ongoing. The [vote on Twitter is in favor of a name change](https://twitter.com/builtbybel/status/1734298143389254043). Interestingly, in the second main poll on my blog, [Bloatynosy is currently leading over Appstrip](https://www.builtbybel.com/blog/bloatynosy-is-now-x#google_vignette). In case you missed it - there is consideration to change the app name. The changes to the core of the application are also substantial, so I am introducing this version a bit more slowly. You can read more about it here.

By the way, I'm still looking for a new icon for the app (whether it's called Appstrip, Bloatynosy, or something else). If anyone would like to contribute, please report it here in the discussions thread. Otherwise, I've seen numerous presentations and tutorials of the application on YouTube and TikTok. If someone is willing to create a brief but professional introduction to the app for the "average" user at home, feel free to express your interest in the discussions.

![yx8k5F0d2D](https://github.com/builtbybel/Bloatynosy/assets/57478606/ae166406-24e2-4044-8d07-f7254ecc0711)

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Bloatynosy/releases/tag/'>Unveiling the New Identity. Bloatynosy or X</a>.</em>

**Comment by ebug64** [comment]
Appreciate you continuing to update this app. However, I don't really like having to add to the list before letting the program just auto-analyze for me now. Don't see why that was changed. Also, I like the new UI, but please add a dark mode already I'm begging you lol
